test/e2e/support/utils.js

```diff
@@ -459,3 +459,19 @@ export async function getAllBlocks() {
 		return select( 'core/editor' ).getBlocks();
 	} );
 }
+
+/**
+ * Binds to the document on page load which throws an error if a `focusout`
+ * event occurs without a related target (i.e. focus loss).
+ */
+export function observeFocusLoss() {
```

Comment:
@aduth - it seems to fail too many test suites. I moved it to utils and applied only for the sidebar. Let's open a follow-up and this assertion where it fits.

Reply:
I also reverted changes applied to `setup-test-framework.js` with https://github.com/WordPress/gutenberg/pull/10917/commits/46a078f397010bb57d8a1049a430bf23ebc6f8ac.

There is this issue that `afterEach` removes page listener but `afterAll` needs them, too.